IO-homework/unitTesting/homework/io/AthleteTest.java

```diff
@@ -0,0 +1,24 @@
+package homework.io;
+
+import org.junit.Test;
+
+import static homework.io.Athlete.shotTimePenalty;
+import static org.junit.Assert.assertEquals;
+
+public class AthleteTest {
+
+   @Test
+   public void testPenalty(){
+       String result = "xoxox";
+       int time = shotTimePenalty(result);
```

Comment:
Testele sunt ok, dar design-ul aplicatiei nu te lasa sa scrii prea multe teste. As mai fi scris un test cu ajutorul caruia sa testez comparatorul clasei `Athlete`(bagi 3 obiecte intro colectie in o anumita ordine, o sortezi si verifici noua ordine).